pkg/windows/windows.go

```diff
@@ -509,6 +512,13 @@ func (vm *windows) ConfigureWICD(apiServerURL string, credentials *Authenticatio
 	return nil
 }
 
+func (vm *windows) deconfigureWICD() error {
```

Comment:
- Please add a comment
- Please move this along with all the other non-struct methods.